tests/framework/extensions/clusters/clusters.go

```diff
@@ -115,13 +115,32 @@ func CheckServiceAccountTokenSecret(client *rancher.Client, clusterName string)
 		logrus.Infof("serviceAccountTokenSecret in this cluster is: %s", cluster.ServiceAccountTokenSecret)
 		return true, nil
 	} else {
-		logrus.Infof("serviceAccountTokenSecret does not exist in this cluster!")
+		logrus.Infof("warning: serviceAccountTokenSecret does not exist in this cluster!")
 		return false, nil
 	}
 }
 
+// Verify if PSACT is enabled or not in the cluster.
```

Comment:
Sorry I missed this can you put CheckPSACT in the comment. Sorry Markus

Reply:
Resolved